edx-platform/pearson-pols-theme/lms/templates/header/navbar-logo-header.html

```diff
@@ -14,7 +14,7 @@
 %>
 
 <h1 class="header-logo">
-  <a href="${branding_api.get_home_url()}">
+  <a href="${configuration_helpers.get_value('LMS_ROOT_URL', settings.LMS_ROOT_URL)}">
```

Comment:
Why not include branding_api.get_home_url() as the default value? Instead of settings.LMS_ROOT_URL @amalbas